File: pset1/mario/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int h;
    do
    {
        printf("Height: ");
        h = get_int();
    }
    while (h < 0 || h > 23);
    
    int r;
    
    for (r = 0; r < h; r++)
        {
            int i;
            
            for (i = 0; i < h - r - 1; i++)
            {
                printf(" ");
            }
 
            int p;
            
            for (p = 0; p < r + 1; p++)
            {
                printf("#");
            }
            
            printf("  ");
            
            for (p = 0; p < r + 1; p++)
            {
                printf("#");
            }
            
            printf("\n");
        }
}

File: pset6/mario.py
import cs50

# Get the height of pyramid
while True:
    print("Height: ")
    h = cs50.get_int()
    if h >= 0 and h <= 23:
        break

for r in range (0, h):
    for i in range(0, h - r -1):
        print(" ", end = "")
    
    for p in range(0, r + 1):
        print("#", end = "")
        
    print("  ", end = "")
    
    for t in range(0, r + 1):
        print("#", end = "")
        
    # leave a line
    print()